deps/popt/patches/0001-remove-_init-_fini-from-exported-symbols.patch

```diff
@@ -0,0 +1,26 @@
+From 283db2c37ae4f81179a6abe1a5369c2dbb84cebe Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo.beauzee@datadoghq.com>
+Date: Thu, 13 Feb 2025 09:27:40 +0100
+Subject: [PATCH] remove _init / _fini from exported symbols
+
+these are deprecated symbols which zig doesn't generate
```

Comment:
I wonder if we should keep patches that target solving "zig-cc" problems separate 🤔 .